demo/app/android/app/src/main/kotlin/walletsdk/flutter/app/MainActivity.kt

```diff
@@ -302,14 +306,47 @@ class MainActivity : FlutterActivity() {
                 val authCodeArgs = call.argument<MutableMap<String, String>>("authCodeArgs")
                     ?: throw java.lang.Exception("authCodeArgs params is missed, Pass scopes, clientID and redirectURI as the arguments")
 
+                var dynamicRegistrationSupported =  openID4CI.dynamicRegistrationSupported()
+                var clientId = authCodeArgs["clientID"].toString()
```

Comment:
Inconsistency in acronym capitalization: you have `clientId` above and `redirectURI` below. Consider changing `clientId` to `clientID` for consistency & readability